src/vs/workbench/browser/parts/editor/noTabsTitleControl.ts

```diff
@@ -198,6 +200,11 @@ export class NoTabsTitleControl extends TitleControl {
 	}
 
 	updateOptions(oldOptions: IEditorPartOptions, newOptions: IEditorPartOptions): void {
+		// Update tab height
```

Comment:
Since this is also called from the tabs title control, we can probably move this into the parent `titleControl.ts` method and then make sure to call `super.updateOptions`